pootle/apps/pootle_score/updater.py

```diff
@@ -6,9 +6,57 @@
 # or later license. See the LICENSE file for a copy of the license and the
 # AUTHORS file for copyright and authorship information.
 
+from datetime import date
+
+from django.utils.functional import cached_property
+
+from pootle.core.delegate import log, event_score
+from pootle_log.utils import LogEvent
+
 
 class StoreScoreUpdater(object):
 
     def __init__(self, store, *args, **kwargs):
         self.store = store
         self.user = kwargs['user']
+
+    @property
+    def event_class(self):
+        return LogEvent
```

Comment:
Just wondering if it is actually needed to use a property to what it looks like a constant value. Is it not possible to just use a class attribute?